README.md

```diff
@@ -1,3 +1,10 @@
+# SC-machine
+
+[//]: <> (TODO replase CodeCov badge links)
+
+[![CI](https://github.com/ostis-dev/sc-machine/actions/workflows/main.yml/badge.svg)](https://github.com/ostis-dev/sc-machine/actions/workflows/main.yml)
+[![codecov](https://codecov.io/gh/fintarin/sc-machine/branch/master/graph/badge.svg?token=QXJOR4YR2O)](https://codecov.io/gh/fintarin/sc-machine)
```

Comment:
Shouldn't use own accounts.

Reply:
I need to know this link for the main project, please tell me it.